mobile/src/people/departments/departments-tree-node.ts

```diff
@@ -2,12 +2,12 @@ import { Employee } from '../../reducers/organization/employee.model';
 import { EmployeeMap } from '../../reducers/organization/employees.reducer';
 import { Department } from '../../reducers/organization/department.model';
 
+export const stubIdForSubordinates = 'subordinates';
```

Comment:
btw, why do you have a stub id at all?? what if i go to database and change one department id to subordinates it just messes up?

Reply:
its used to differentiate department's head card and lowest level employee card in horizontal scroll card set.

DB id would mess it up - that's right :)